tracim/tracim/lib/webdav/sql_resources.py

```diff
@@ -1005,6 +1005,40 @@ def move_file(self, destpath):
 
         transaction.commit()
 
+    def copyMoveSingle(self, destpath, isMove):
+        if isMove:
+            raise NotImplemented
```

Comment:
Do you know why wsgidan provide a "copy move" function ? it's weird. No risk than that method called for move something ?

Reply:
I do think it's because move and copy are really similar. As far as we override move_recursive method, there should not be case where this method is used for move. I should check this. Normally instead of NotImplemented, call existing move_file method should correct this case if it exist, but as far as i didn't see this case, i prefer to do it.